Gruppe.A1.End-to-End_Encryption_mit_IPv6_und_integriertem_IPSec.ThushjandanPonnudurai.PascalBaumann.Termpaper.tex

```diff
@@ -111,19 +112,104 @@ \subsection{IPsec}
 
 The next consideration concerns the key management. Although manual key management can be used, it's generally not advised and disables the replay detection mechanism.The most widely used key management is the Internet Key Exchange (IKE), which is available in its original form (IKEv1) and in a new, simpler version (IKEv2). In network environment where mainly Microsoft systems are used, the usage of Kerberised Internet Negotiation of Keys (KINK) is available, as this depends on the Kerberos system. \parencite[4-5]{Bellovin2009}
 
-\section{IPsec}
-\label{sec:IPsec}
+\section{ISAKMP}
+\label{sec:ISAKMP}
 
-\subsection{Algorithmen \& DHs}
+\subsection{Algorithms \& DHs}
 \label{ssec:AlgoDH}
+Data confidentiality is the key requirement for any VPN. The confidentiality of the data is provided by the encryption algorithm. That's why it is very important to choose the optimal encryption algorithm. \parencite{Snader2006}
```

Comment:
Please add a paragraph why an encryption algorithm is secure, e.g. mention the cost of an attack. Why you have to choose a long password, what a strong password is, strong PSK etc.